.sample.env

```diff
@@ -1,3 +1,3 @@
-# http://ddollar.github.com/foreman/
-RACK_ENV=development
-SECRET_KEY_BASE=development_secret
+export RACK_ENV=development
+export SECRET_KEY_BASE=development_secret
+export PORT=3000
```

Comment:
Why this change? This file is intended for Foreman, for which the format was correct.

Reply:
I think I changed it so I can run `source .env` and run processes without foreman if I wanted. No big deal, happy to revert it